infrastructure/gsuite-exporter/gsuite_exporter/cli.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#            http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import importlib
+import re
+import sys
+import logging
+from gsuite_exporter import exporters
+from gsuite_exporter.collectors.reports import AdminReportsAPIFetcher
+
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+logging.getLogger('googleapiclient').setLevel(logging.ERROR)
+
+logger = logging.getLogger(__name__)
+
+def get_exporter_cls(exporter_class):
+    try:
+        file, name = tuple(exporter_class.split('.'))
```

Comment:
Why cast this to a tuple?  The list returned from the split will unpack just as well.

Additionally, `file` shadows builtin function.

Reply:
Same comment as before.  `file` shadows built-in, so maybe use `file_`, which is a common convention and mentioned in pep8.  The tuple call is unnecessary as the list will unpack.